Makefile

```diff
@@ -97,12 +97,22 @@ vet:
 			echo "and fix them if necessary before submitting the code for reviewal."; \
 		fi
 
+ifeq ($(OS),Windows_NT)
+INSTALL_GIT_HOOKS := cp .hooks/* .git/hooks/
+else
+INSTALL_GIT_HOOKS := for H in .hooks/*; do ln -sf ../../$$H .git/hooks/; done
+endif
+
+githooks:
+	@if [ -d .git/hooks ]; then $(INSTALL_GIT_HOOKS); fi
+
```

Comment:
Rm extra lines?

Reply:
Done

Reply:
Interestingly, this review comment is still not out of date, because GitHub cannot distinguish between the two lines I removed (which included this one) and the single line I left in place (which was 2 lines below) :P comon GitHub, you still can't read my mind?!